zigate/clusters.py

```diff
@@ -311,6 +327,18 @@ def __init__(self, endpoint=None):
                                             'expire': 2, 'type': int},
                                    }
 
+        #lumi.remote.b286acn01
+        wallswitch_b286acn01 = {
+          0x05F01 : 'right',
+          0x05F02 : 'left',
+          0x05F03 : 'both'
+        }
+        if self._endpoint['device'] in wallswitch_b286acn01:  # lumi.remote.b1acn01
+            self.attributes_def = {0x0055: {'name': wallswitch_b286acn01[self._endpoint['device']],
+                                            'value': 'xiaomi_wireless_wallswitch_decode(value)',
+                                            'expire': 0.5, 'expire_value': '', 'type': str},
```

Comment:
0.5 is too fast, 1 or 2 sec looks better
```suggestion
                                            'expire': 2, 'expire_value': '', 'type': str},
```